<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioRecordForm</class>
 <widget class="QWidget" name="AudioRecordForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>输入设备</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>目的地址</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>目的端口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>采样率</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>声道个数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>采样大小</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>采样类型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>字节序</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QComboBox" name="cmbAudioDevices"/>
     </item>
     <item>
      <widget class="QLineEdit" name="leHostAddr">
       <property name="text">
        <string>192.168.1.104</string>
       </property>
       <property name="placeholderText">
        <string>主机地址</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leHostPort">
       <property name="text">
        <string>9000</string>
       </property>
       <property name="placeholderText">
        <string>主机端口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbSampleRate"/>
     </item>
     <item>
      <widget class="QComboBox" name="cmbChCnt"/>
     </item>
     <item>
      <widget class="QComboBox" name="cmbSampleSize"/>
     </item>
     <item>
      <widget class="QComboBox" name="cmbSampleType"/>
     </item>
     <item>
      <widget class="QComboBox" name="cmbByteOrder"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>162</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="btnRecord">
     <property name="text">
      <string>Start Record</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
